### DISCUSSION TITLE
Fees in summary

### DISCUSSION THREAD

**Discussion by Dilosen** [ghostfolio/ghostfolio#3559] [Q&A] [answered]
The fees displayed in the summary is displaying incorrectly. It is showing in ZAR (base currency) but is imported in ZAc (denomination).

Thus, the value is 100x the value it should be, i.e R10 is showing as R1000

**Comment by dtslvr** [comment]
Hi @Dilosen

Could you please create a bug report including detailed steps to reproduce the problem and the expected behavior. Additionally, please attach an exported activity. Thanks for your assistance.

**Comment by Dilosen** [comment]
Hi @dtslvr 

To recreate the issue: 

1. Import mulitple activites for any account
2. navigate to the overview page -> summary
3. In the Investment section, fees is an incorrect calculation

The miscalculation only occurs in the summary page in the overview section. The fee amount displayed is equivelent to  40% of the total purchases, and should be far, far less. I believe it's recording the value by multiplying by 100.  The fee value is in ZAR; the base currency for the account. 

For each activity in any account, it is correctly recording the ZAc fees, totaling it and then converting it to ZAR in the 'company overview'. Therefore, I can see the fees are correctly calculating as well as other totals for quanitity etc. 

My assumption is that when it is calculating the total in the summary section, it's multiplying by 100 again, when it already did it in the totaling calculation/conversion.

**Comment by dtslvr** [comment, marked as answer]
Pull request: https://github.com/ghostfolio/ghostfolio/pull/3567

**Reply by Dilosen** [reply]
Issue is resolved in the latest update.
